project1/models.py

```diff
@@ -27,4 +35,34 @@ def __init__(self, username, email, password) :
         self.created = dt.now()
 
     def __repr__(self):
-        return self.email
\ No newline at end of file
+        return self.email
+
+'''
+Class for storing book details
+'''
+class Book(db.Model):
+    __tablename__ = "books"
+    id = db.Column(db.Integer,
```

Comment:
We can directly use ISBN as primary key instead of adding another variable.

Reply:
yes, we can do that Ashish. As of now, I thought of using some index to show the list of books.

Reply:
But choosing the primary key as id might not be the best thing to do. In future we might need to link one table to another table where id wouldn't serve the purpose I guess.

Reply:
Yes, I think you are right.